src/parser/jobs/mnk/modules/Chakra.tsx

```diff
@@ -0,0 +1,137 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import Color from 'color'
+import {DataLink} from 'components/ui/DbLink'
+import {ActionKey} from 'data/ACTIONS'
+import {JOBS} from 'data/JOBS'
+import {Event, Events} from 'event'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge} from 'parser/core/modules/Gauge'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const CHAKRA_CAP = 5
+const BH_CHAKRA_CAP = 10
+
+const CHAKRA_OVERCAP_SEVERITY = {
+	1: SEVERITY.MINOR,
+	5: SEVERITY.MEDIUM,
+	8: SEVERITY.MAJOR,
+}
+
+const ALLOWED_GCDS_WITH_CHAKRA: ActionKey[] = [
+	'FORM_SHIFT',
+	'SIX_SIDED_STAR', // 6SS spends chakra when used
+]
+
+/** Graph colors */
+const FADE_AMOUNT = 0.25
+const CHAKRA_COLOUR = Color(JOBS.MONK.colour).fade(FADE_AMOUNT)
+
+export class Chakra extends Gauge {
+	static override handle = 'chakra'
+	static override title = t('mnk.gauge.chakra.title')`Chakra`
+
+	@dependency private suggestions!: Suggestions
+
+	private chakraGauge: CounterGauge | null = null
+	private allowedGcdIds = ALLOWED_GCDS_WITH_CHAKRA.map(actionKey => this.data.actions[actionKey].id)
+	private gcdsUsedAtChakraCap = 0
+
+	override initialise() {
+		super.initialise()
+
+		const gaugeFilter = filter<Event>()
+			.actor(this.parser.actor.id)
+			.type('gaugeUpdate')
+
+		this.addEventHook(gaugeFilter, this.onGaugeUpdate)
+	}
+
+	private isDamagingGcd = (id: number): id is number => {
+		const action = this.data.getAction(id)
+		return !!(action && action.onGcd && !this.allowedGcdIds.includes(id))
+	}
+
+	private initialiseGauge() {
+		this.parser.actor.loggedGauge = true
+
+		this.chakraGauge = this.add(new CounterGauge({
+			maximum: CHAKRA_CAP,
+			graph: {
+				label: <Trans id="mnk.gauge.chakra.label">Chakra</Trans>,
+				handle: 'chakra',
+				height: 32,
+				color: CHAKRA_COLOUR,
+				collapse: false,
```

Comment:
Minor nit: The timeline row header ends up looking a bit weird because you're not collapsing it:
![image](https://github.com/user-attachments/assets/c9142821-7296-4517-9758-78c611e67e7b)

I would use `forceCollapse: true` instead of `collapse: false` for this one. This one functions more like the Nadi Gauge than the Fury Gauge, so it'll end up displaying the row header like so:
![image](https://github.com/user-attachments/assets/e1520456-380c-41e3-a0b1-20ef11bbcdd2)